doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -137,8 +137,8 @@ The Infrastructure Profile Catalog contains the following attributes:
   * Acceleration capabilities
 * Profiles are offered to VNFs as an instance types with predefined compute flavors.
   * A particular set of options is an instance type
-  * Compute flavors: S, M, L
-* VNF performance profiles, for which NFVI validations will support and be verified against, are defined as basic, network intensive, and compute intensive. Details for each of these profiles can be found in chapter 2.3.
+  * Compute flavors: .tiny, .small etc as defined in [RM §4.2.1.1](../../../ref_model/chapters/chapter04.md#4211-predefined-compute-flavours)
+* NFVI performance profiles, for which NFVI validations will support and be verified against, are defined as basic and network intensive. <!-- , and compute intensive. --> Details for each of these profiles can be found in chapter 2.3.
 <p align="center"><img src="../figures/RC_NFVI_Profiles.png" alt="NFVI Profiles" title="NFVI Profiles" width="100%"/></p>
```

Comment:
@mf4716 Do you have image source used here to create png? Couple of diagrams still show compute intensive profiles.

Reply:
@deebhatia diagrams should be [here](https://github.com/cntt-n/CNTT/blob/master/artefacts/CNTT_diagrams.pptx)

Reply:
thanks @CsatariGergely.